src/Bones/babylon.bone.ts

```diff
@@ -551,26 +612,41 @@
 
         }
 
-        public getRotation(mesh: AbstractMesh): Quaternion {
+        public getRotation(space = Space.LOCAL, mesh?: AbstractMesh): Quaternion {
 
             var result = Quaternion.Identity();
 
-            this.getRotationToRef(mesh, result);
+            this.getRotationToRef(result, space, mesh);
 
             return result;
 
         }
 
-        public getRotationToRef(mesh: AbstractMesh, result: Quaternion): void{
+        public getRotationToRef( result: Quaternion, space = Space.LOCAL, mesh?: AbstractMesh): void{
+

```

Comment:
I tried this with Bone.getDirectionToRef:

getDirectionToRef(localAxis, mesh, result);

Typescript wouldn't allow it though due to Node.getDirectionToRef(localAxis, result).